README.md

```diff
@@ -5,3 +5,64 @@ Runs the query using different scheduling techniques and batch sizes
 ```sql
 select string_optional from t where dict_10_required = 'prefix#1' and dict_1000_required = 'prefix#1';
 ```
+
+
+## Setup:
+1. Install git-lfs
+2. Run `git lfs pull` to pull files
```

Comment:
This shouldn't be necessary if git LFS was installed prior to checking out the repo

Reply:
Ah -- I did not have `lfs` installed at that point

```suggestion
2. Run `git lfs pull` to pull files (not needed if LFS was installed prior to checking out the repo)
```